### PR TITLE
feat: use fqin in from

### PR DESCRIPTION
**Pull request by Avi98**
fixes [#442](https://github.com/hedgedoc/container/issues/422)

### CODE REVIEW

File: alpine/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM --platform=$BUILDPLATFORM node:20.2.0-alpine@sha256:59ac6536ba03469adc3847f23a4f223b0418fba21c90168d703f61bd84125989 AS builder
+FROM --platform=$BUILDPLATFORM docker.io/library/node:16.20.0-alpine@sha256:b4a72f83f52bbe3970bb74a15e44ec4cf6e873ad4787473dfc8a26f5b4e29dd2 AS builder
```
Comment (SISheogorath) [inline alpine/Dockerfile:1]: This looks like a substantial downgrade from node 20 to node 16? (and it seems to persist throughout the change)

Comment (Avi98) [reply alpine/Dockerfile]: my bad @SISheogorath  i have changed to node 20

**Review by SISheogorath — APPROVED** [review]
LGTM, thanks a lot for your work!

**Comment by mrdrogdrog** [comment]
Rebase

**Review by mrdrogdrog — APPROVED** [review]

**Pull request merged**